app/javascript/controllers/keywords_controller.js

```diff
@@ -29,6 +29,13 @@ export default class extends Controller {
     this.containerTarget.classList.remove('is-invalid')
   }
 
+  checkForEnter(event) {
+    if (event.keyCode === 13) {
```

Comment:
Would you consider to changing this to `event.key === 'Enter'`? I read earlier today that `.keyCode` is deprecated.

Reply:
Done. Thanks for the pointer.